Source/VisualStudioTools/Private/VSServerCommandlet.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 (c) Microsoft. All rights reserved.
+
+#include "VSServerCommandlet.h"
+
+#include "HAL/PlatformNamedPipe.h"
+#include "Runtime/Core/Public/Async/TaskGraphInterfaces.h"
+#include "Runtime/Core/Public/Containers/Ticker.h"
+#include "Runtime/Engine/Classes/Engine/World.h"
+#include "Runtime/Engine/Public/TimerManager.h"
+#include "Runtime/Launch/Resources/Version.h"
+#include <chrono>
+#include <codecvt>
+#include <fstream>
+#include <string>
+#include <thread>
+#include <windows.h>
+
+#include "VisualStudioTools.h"
+
+static constexpr auto NamedPipe = TEXT("namedpipe");
+FString UEServerNamedPipe;
```

Comment:
This one is global but there's also a same named member variable. Keep the member variable one?